cfg/rippled-example.cfg

```diff
@@ -1281,7 +1281,7 @@
 #       default. Don't change this without understanding the consequences.
 #
 #       Example:
-#           gas_price = 2000000      # 2 drops per gas
+#           gas_price = 1000000      # 1 drop per gas
```

Comment:
I vote for lower the price. Olek's base58 conversion example took 200k gas, so 0.2 XRP, about $0.45. seems high.